_content/tour/rus/welcome.article

```diff
@@ -1,211 +1,198 @@
+знания языка программирования с большим акцентом на механике языка, философии дизайна и рекомендациях. Мы сосредотачиваемся на обучении, как писать код с приоритетом на согласованность, целостность, читаемость и простоту. Мы многое обсуждаем в контексте "если производительность важна", с акцентом на механической симпатии, дата-ориентированном дизайне, разделении и написании/отладке программного обеспечения.
 
-** Our Teacher
+** Наш учитель
 
-William Kennedy is a managing partner at Ardan Labs in Miami, Florida. Ardan Labs
-is a high-performance development and training firm working with startups and
-fortune 500 companies. He is also a co-author of the book Go in Action, the author
-of the blog GoingGo.Net, and a founding member of GoBridge which is working to
-increase Go adoption through diversity.
+Уильям Кеннеди - управляющий партнер Ardan Labs в Майами, Флорида. Ardan Labs
+является фирмой по разработке и обучению высокой производительности, работающей со стартапами и
+компаниями Fortune 500. Он также является соавтором книги "Go in Action", автором
+блога GoingGo.Net и одним из основателей GoBridge, который работает над
+увеличением принятия Go через разнообразие.
 
-*Video* *Training*
+Обучающие видео
 
 - [[https://education.ardanlabs.com/][Ultimate Go Video]]
 - [[http://youtube.ardanlabs.com/][Ardan Labs YouTube Channel]]
 
-*Blog*
-
-- [[https://www.ardanlabs.com/blog/][Ardan Labs Technical Blog]]
+*Блог*
 
-*Podcasts*
+- [[https://www.ardanlabs.com/blog/][Технический блог Ardan Labs]]
 
-- [[https://ardanlabs.buzzsprout.com/][Ardan Labs Podcast: On Going Series]]
+*Подкасты*
 
-** More About Go
+- [[https://ardanlabs.buzzsprout.com/][Подкаст Ardan Labs: серия On Going]]
+** Еще о Go
 
-Go is an open source programming language that makes it easy to build simple,
-reliable, and efficient software. Although it borrows ideas from existing languages,
-it has a unique and simple nature that make Go programs different in character from
-programs written in other languages. It balances the capabilities of a low-level
-systems language with some high-level features you see in modern languages today.
-This creates a programming environment that allows you to be incredibly productive,
-performant and fully in control; in Go, you can write less code and do so much more.
+Go - это язык программирования с открытым исходным кодом, который облегчает создание простого,
+надежного и эффективного программного обеспечения. Несмотря на то, что он заимствует идеи из существующих языков,
+у него есть уникальная и простая природа, которая отличает программы на Go от программ, написанных на других языках.
+Он совмещает возможности низкоуровневого системного языка с некоторыми высокоуровневыми функциями,
+которые вы видите в современных языках. Это создает программную среду, которая позволяет вам быть невероятно продуктивным, производительным и полностью контролируемым; на Go вы можете писать меньше кода и делать намного больше.
```

Comment:
```suggestion
Go — язык программирования с открытым исходным кодом, который позволяет легко создавать
простое, надежное и эффективное программное обеспечение. Хотя он заимствует идеи
из существующих языков, он имеет уникальную и простую природу, которая делает
программы Go отличными от программ, написанных на других языках. Он балансирует
возможности низкоуровневого системного языка с некоторыми высокоуровневыми
возможностями, которые вы видите в современных языках. Это создает программную
среду, которая позволяет вам быть невероятно продуктивным, производительным и
полностью контролировать; в Go вы можете писать меньше кода и делать гораздо больше.
```

I translated this part in a slightly different way, but please feel free to use your translation if you like it better.